website/views/core.py

```
@@ -2923,3 +2928,122 @@ def get_context_data(self, **kwargs):
+        referral_link = context.get("referral_link", default_referral_url)
+
+        try:
+            email_body = render_to_string(
+                "email/organization_invite.html",
+                {
+                    "org_name": org_name,
+                    "referral_link": referral_link,
+                    "sender_name": sender_name,
+                },
+            )
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)